src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -695,6 +698,64 @@ class spell_mage_water_elemental_freeze : public SpellScript
     }
 };
 
+/* 112965 - Finger of Frost
+   228597 - Frostbolt
+   84721  - Frozen Orb */
+class spell_mage_finger_of_frost : public SpellScriptLoader
+{
+public:
+    spell_mage_finger_of_frost() : SpellScriptLoader("spell_mage_finger_of_frost") { }
+
+    class spell_mage_finger_of_frost_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_mage_finger_of_frost_SpellScript);
+
+        bool Validate(SpellInfo const* spellInfo) override
```

Comment:
```suggestion
        bool Validate(SpellInfo const* /*spellInfo*/) override
```
Need to comment out unused arguments to pass the compiler warnings (treated as error).